src/psyclone/tests/psyir/nodes/loop_test.py

```diff
@@ -59,15 +59,7 @@ def test_loop_init():
     '''
     loop = Loop()
     assert loop.parent is None
```

Comment:
pylint says `get_invoke` is imported but unused in this file.

Reply:
Done